corehq/apps/accounting/models.py

```diff
@@ -679,12 +679,9 @@ def new_rate(cls, feature_name, feature_type,
         feature, _ = Feature.objects.get_or_create(name=feature_name, feature_type=feature_type)
         rate = FeatureRate(feature=feature)
 
-        if monthly_fee is not None:
-            rate.monthly_fee = monthly_fee
-        if monthly_limit is not None:
-            rate.monthly_limit = monthly_limit
-        if per_excess_fee is not None:
-            rate.per_excess_fee = per_excess_fee
+        rate.monthly_fee = monthly_fee or rate.monthly_fee
```

Comment:
Aren't these [decimals and integers](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/accounting/models.py#L649-L655)?